functional-tests/README.md

```diff
@@ -33,6 +33,10 @@ We use Poetry for managing the test dependencies.
 ```bash
 # install poetry via pip3
 pip3 install poetry
```

Comment:
This doesn't work as expected on Debian based distros now.  Should be `apt install python3-poetry` to install the system version.

Reply:
Good catch! Added